File: src/main/java/um/haberes/core/model/view/NovedadExiste.java
package um.haberes.core.model.view;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import org.hibernate.annotations.Immutable;

@Entity
@Table(name = "vw_novedad_existe")
@Immutable
@Getter
@Setter
@EqualsAndHashCode
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class NovedadExiste {

    @Id
    private String uniqueId = null;

    private Long legajoId = null;

    private Integer anho = null;

    private Integer mes = null;
}

File: src/main/java/um/haberes/core/kotlin/model/view/NovedadExiste.kt
package um.haberes.core.kotlin.model.view

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import org.hibernate.annotations.Immutable

@Entity
@Table(name = "vw_novedad_existe")
@Immutable
data class NovedadExiste(

    @Id
    var uniqueId: String? = null,

    var legajoId: Long? = null,
    var anho: Int? = null,
    var mes: Int? = null

)
